src/main/java/com/fasterxml/jackson/core/Base64Variant.java

```diff
@@ -87,6 +87,11 @@
      */
     private final transient boolean _usesPadding;
 
+    /**
+     * Whether padding characters should be required or not while decoding
+     */
+    private final transient PaddingReadBehaviour _paddingReadBehaviour;
```

Comment:
Let's remove `transient` here -- not sure why preceding final field has it, possibly copy-paste issue originally (I will remove it from others too)

Reply:
Removed transient